charts/jenkins/templates/_helpers.tpl

```diff
@@ -206,6 +207,94 @@ jenkins:
         {{- end }}
       {{- end }}
       {{- end }}
+    {{- if .Values.additionalClouds }}
```

Comment:
could you add a unit test for this please?
https://github.com/jenkinsci/helm-charts/blob/main/CONTRIBUTING.md#unit-tests